### DISCUSSION TITLE
Subtasks should allow for tag assignment

### DISCUSSION THREAD

**Discussion by jmiodownik** [super-productivity/super-productivity#2234] [Ideas]
Currently subtasks do not seem to support the assignment of tags (example: Today). It would be helpful to categorize subtasks uniquely from their parent and additionally display them in the corresponding filtered overviews, such as the Today view

**Comment by fccfc5d245c** [comment]
This is the feature I need most. I'm using this software in weird ways to work around not categorizing these tasks.

**Reply by johannesjo** [reply]
I completely agree! This would be nice to have.

**Comment by Jagdfalke** [comment]
Now, there is an issue regarding this idea: #2946
